Cargo.toml

```diff
@@ -88,7 +93,7 @@ members = [
 [workspace.dependencies]
 # `git`-based deps
 hif = { git = "https://github.com/oxidecomputer/hif" }
-humpty = { git = "https://github.com/oxidecomputer/humpty", version = "0.1.1" }
+humpty = { git = "https://github.com/oxidecomputer/humpty", branch = "single-task-dump", version = "0.1.2" }
```

Comment:
Just a reminder to move this over before pushing.

Reply:
Done!